### ISSUE TITLE
Allow any binary operator to be used as an updater

### ISSUE THREAD

**Issue by cao-** [JuliaLang/julia#47899]
Some binary operators can be used as an "updater" operator, by appending the `=` sign.  For instance we have `+=`, `-=` or `&=`.  But not all binary operators have an updater version, like the union operator can't be used as `∪=`.  Wouldn't it be nice if this worked for all binary operators?

**Comment by vtjnash** [comment]
Duplicate of https://github.com/JuliaLang/julia/issues/46371

**Comment by StefanKarpinski** [comment]
I don't think this is a duplicate. The linked issue seems to be about having an updating form for arbitrary functions whereas this is just about having updating forms for more infix binary operators.

**Comment by mbauman** [comment]
Well, now a future duplicate of this issue has far surpassed the discussion and thoughts here.  Seems sensible to focus things in [#56821](https://github.com/JuliaLang/julia/issues/56821).
